BruteShark/PcapAnalyzer/Modules/NetworkMapModule/Neo4jJsonExporter.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.IO;
+
+namespace PcapAnalyzer
+{
+    public class Neo4jJsonExporter
+    {
+
+        public static string GetNetworkMapAsJsonString(List<NetworkConnection> connections)
+        {
+            var options = new JsonSerializerOptions
```

Comment:
object initializer will be more elegant, all this lines can be a one liner with less initializations